lib/iris/plot.py

```diff
@@ -37,14 +37,14 @@
 import matplotlib.pyplot as plt
 import matplotlib.ticker as mpl_ticker
 from mpl_toolkits.axes_grid.anchored_artists import AnchoredText
-import nc_time_axis
 import netcdftime
```

Comment:
But I think it's not worth it, as netcdf is a required dependency, ***not*** optional
(and it's just part of that)

Plus, it has nothing to do with the issue in hand.